src/style.css

```diff
@@ -0,0 +1,106 @@
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.container {
+  width: 200px;
+  display: flex;
+  flex-wrap: wrap;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.imac {
+  height: 194px;
+}
+
+.imac img {
```

Comment:
Try to avoid stylizing tags. Always use classes for this. Tags stylizing is OK when you reset default browser styles, like margin: 0 for BODY tag, which has 8px margins.

Reply:
Also try to avoid increasing the specificity of the selector. One class is enough to identify the elements on the page. No need to write something like .block.colored-block. Use .block and .block--colored instead, and apply each of them to your element

Reply:
fix this